partner_private_information/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Roger Sans <roger.sans@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Partner Private Information",
+    "version": "17.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Sygel, Odoo Community Association (OCA)",
+    "category": "Contacts",
+    "summary": "Add private information tab in partners.",
+    "website": "https://github.com/sygel-technology/sy-partner-contact",
+    "depends": [
+        "base",
```

Comment:
Please, remove this dependency as contacts module already depends on base module

Reply:
Done